src/createQueryGroupTags.ts

```diff
@@ -1,7 +1,8 @@
-import { QueryTag, QueryTagKey } from "./types/tags"
+import { QueryTag } from "./types/tags"
+import { TagKey } from "./getTagKey"
 
 export function createQueryGroupTags() {
-  const tags = new Map<QueryTagKey, Set<number>>()
+  const tags = new Map<TagKey, Set<number>>()
```

Comment:
this was moved and renamed. Open to keeping `Query` in the name if you feel like it provides value